cloudify_cli/cli/cfy.py

```diff
@@ -155,6 +159,29 @@ def inputs_callback(ctx, param, value):
     return inputs_to_dict(value)
 
 
+def _get_execution_id(ctx, param, value):
+    tenant = ctx.params.get('tenant_name')
+    is_uuid = value and re.match('[a-fA-F0-9-]{30,40}', value)
```

Comment:
Wouldn't it be better to try to select it from the execution IDs?

Reply:
don't really want to add an additional request